data/115/885/724/5/1158857245.geojson

```diff
@@ -6,17 +6,23 @@
     "edtf:inception":"uuuu",
     "gbr-datalondon:GSS_CODE":"E09000033",
     "gbr-datalondon:ONS_INNER":"T",
-    "geom:area":0.002783,
-    "geom:area_square_m":21415552.396144,
-    "geom:bbox":"-0.216088,51.484801,-0.111101,51.539793",
-    "geom:latitude":51.513899,
-    "geom:longitude":-0.161304,
+    "geom:area":0.002854,
+    "geom:area_square_m":21965353.48086,
+    "geom:bbox":"-0.216066679994,51.483807039,-0.111057191927,51.5398141282",
+    "geom:latitude":51.513445,
+    "geom:longitude":-0.160439,
     "iso:country":"GB",
-    "lbl:latitude":51.497991,
-    "lbl:longitude":-0.12875,
+    "label:eng_x_preferred_longname":[
+        "City of Westminster London Boro"
```

Comment:
Looks odd.. should this be "borough"? Should the longname include all of this?

Reply:
This is coming directly from the Ordnance Survey, so....?